lessons/basics/comprehensions.md

```diff
@@ -105,7 +105,7 @@ iex> for {k, v} <- [one: 1, two: 2, three: 3], into: %{}, do: {k, v}
 %{one: 1, three: 3, two: 2}
 ```
 
-Since bitstrings are enumerable we can use list comprehensions and `:into` to create strings:
+Since binaries are collectable we can use list comprehensions and `:into` to create strings:
```

Comment:
the information about is correct, but there was way to do it directly within Elixir.

```elixir
iex(1)> path = :code.lib_dir(:elixir, :ebin)

# modules that implement the Collectable protocol
iex(2)> Protocol.extract_impls(Collectable, [path]) |> Enum.sort
[BitString, File.Stream, HashDict, HashSet, IO.Stream, List, Map, MapSet]


# modules that implement the Enumerable protocol
iex(3)> Protocol.extract_impls(Enumerable, [path]) |> Enum.sort 
[File.Stream, Function, GenEvent.Stream, HashDict, HashSet, IO.Stream, List,
 Map, MapSet, Range, Stream]
```